tests/guilds/members/avatarUrl.test.ts

```diff
@@ -0,0 +1,61 @@
+import { loadBot } from "../../mod.ts";
+import { assertEquals } from "../../deps.ts";
+Deno.test("[member] avatar URL test group", async (t) => {
+  const bot = loadBot();
+  await t.step("[member] avatar URL user default avatar", async () => {
+    assertEquals(bot.helpers.avatarURL(379643682984296448n, "2195"), "https://cdn.discordapp.com/embed/avatars/0.png");
+  });
+  await t.step("[member] avatar URL user avatar", async () => {
```

Comment:
The tests after this do not depend on this step being completed so this is not an ideal way of doing it. Either move all these tests to the same style as utils.test.ts(because these tests do not make requests to the api) or you can promise.all these steps which im not really a fan of